### PR TITLE
Hw5 git,dom,oop

### PR DESCRIPTION
**Pull request by TG721**

### CODE REVIEW

**Comment by TG721** [comment]
regarding force-pushing the HW5 branch from a488f3a to 1ba95ba. I did that to rename last commit which had a word misspelled in its comment.

File: HM5/functional/functional.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+(function () {
```
Comment (pluhmylkin) [inline HM5/functional/functional.js:1]: please remove unused code

File: HM5/oop/app.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+const imgVar = document.createElement('img');
+imgVar.addEventListener("click", function(event) {
+    const parentElement = imgVar.parentElement;
+    const index = Array.from(parentElement.parentElement.children).indexOf(imgVar);
+    allTasks.removeItemAt(index);
+    parentElement.remove();
+  });
+imgVar.src = "images/trash.svg";
+imgVar.className = "imgButton";
+imgVar.style.marginLeft = "100px";
```
Comment (pluhmylkin) [inline HM5/oop/app.js:107]: please use the css

File: HM5/oop/components.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    }
+}
+
+class Input extends Component {
+    constructor(props) {  //{placeholder, width ...} 
+        super(props);
+        this.element = document.createElement('input');
+        this.element.style.color = 'black';
+        this.element.style.backgroundColor = '#F5F5F5';
+        this.element.style.borderRadius = '10px';
```
Comment (pluhmylkin) [inline HM5/oop/components.js:14]: please use the css

File: HM5/oop/components.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+}
+
+class Modal extends Component {
+    constructor(props) {   //{children}
+        super(props);
+        this.element.style.background = "white";
+        this.element.style.position = "fixed";
+        this.element.style.width = "524px";
+        this.element.style.height = "371px";
+        this.element.style.border = "1px solid #D2D2D2";
```
Comment (pluhmylkin) [inline HM5/oop/components.js:76]: please use the css

File: HM5/oop/app.js [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+      console.log("input text: ", inputText, " taskTitle: ", taskTitle)
+      if((taskTitle.toLowerCase()).includes((inputText).toLowerCase())) currentTask.style.display = "block"
+      else currentTask.style.display = "none"
+    }
+
+  });
+const newTaskButton = new Button({text:"+ New Task", color:"#0053CF", bgColor:"#3C86F426", width:"15%", class:"button",   onClick: function() {
+    addTaskButton.setState({bgColor: "#D3D3D3"}); //make button color grey 
+    modalInput.setState({isEmpty: true});         //empty input field 
+    modal.setState({display: "block"});           //show modal
```
Comment (pluhmylkin) [inline HM5/oop/app.js:20]: style in state it's new for me.

Yes, it works, but for style better to use css

**Review by pluhmylkin — APPROVED** [review]

**Pull request merged**